app/views/users/index.html.erb

```diff
@@ -10,7 +10,9 @@
   <% @users.each do |user| %>
     <tr>
       <td><%= user.email %></td>
-      <td><%= button_to '+', user_contact_requests_path(user), method: :post, id: user.id %></td>
+      <% if not current_user.sent_contact_request(user) %>
```

Comment:
If you only use the method this way, why didn't you add a method like `no_contact_to?`? The `not` would not be necessary then (and besides, RuboCop would prefer `!` instead of `not`)

Reply:
it is not the latest version but thanks for the advice!

Reply:
I don't think `no_contact_to?` would be a good name, since it is not only the persons you have already as contacts but also the ones you only sent a request to

Reply:
Agreed, naming's always difficult :sweat_smile: The point was that the method could return a boolean that does not have to be inverted.

Reply:
A better proposition: use `unless` instead of `if`

Reply:
is in the current code not aplicable anymore